plugins/outputs/rabbitmq/rabbitmq.go

```diff
@@ -0,0 +1,131 @@
+package rabbitmq
+
+import (
+	"fmt"
+	"log"
+	"sync"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"github.com/influxdata/telegraf/plugins/serializers"
+	"github.com/streadway/amqp"
+)
+
+// RabbitMQ is the top level object for this output plugin
+type RabbitMQ struct {
+	Username     string
+	Password     string
+	RabbitmqHost string
+	RabbitmqPort string
+	Queue        string
+
+	sync.Mutex
+
+	serializer serializers.Serializer
+	conn       *amqp.Connection
+	ch         *amqp.Channel
+	q          amqp.Queue
+}
+
+func (rmq *RabbitMQ) publish(body string) error {
+	err := rmq.ch.Publish("", rmq.q.Name, false, false, amqp.Publishing{
```

Comment:
"Since publishings are asynchronous, any undeliverable message will get returned by the server. Add a listener with Channel.NotifyReturn to handle any undeliverable message when calling publish with either the mandatory or immediate parameters as true."

https://godoc.org/github.com/streadway/amqp#Channel.Publish, do you have handle for returned message.

I think these defaults should declared as const or configuration